src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
```

Comment:
The message in the `doWork` method should indicate the type of `Machine` that is working, which is correct here as it mentions `Excavator`.